sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/RegexpExpressionsSuite.scala

```diff
@@ -51,71 +51,123 @@ class RegexpExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
 
     // null handling
     checkLiteralRow(Literal.create(null, StringType).like(_), "a", null)
+    checkLiteralRow(Literal.create(null, StringType).like(_, "/"), "a", null)
```

Comment:
Why we need these tests?

Reply:
I think these tests could prove that the behavior is correct after support escape char.

Reply:
plz drop these tests and add a new test case for various escape characters?

Reply:
OK. I add a new test case.